templates/components/multiagent/typescript/nextjs/route.ts

```diff
@@ -41,9 +44,10 @@ export async function POST(request: NextRequest) {
     });
     const { stream, dataStream } =
       await createStreamFromWorkflowContext(context);
-
-    // Return the two streams in one response
-    return new StreamingTextResponse(stream, {}, dataStream);
+    const streamIterable = streamToAsyncIterable(stream);
+    return LlamaIndexAdapter.toDataStreamResponse(streamIterable, {
+      data: dataStream,
+    });
```

Comment:
how about calling `streamToAsyncIterable` inside of `createStreamFromWorkflowContext`
```suggestion
    return LlamaIndexAdapter.toDataStreamResponse(stream, {
      data: dataStream,
    });
```